packages/settings-library/src/settings_library/_constants.py

```diff
@@ -2,5 +2,7 @@
 
 GB = 1024**3
 
+MINUTE = 60
```

Comment:
THOUGHT: constants that help readability (like MINUTE here) are very handy but have the danger that you need to have a clear understanding of the reference value. E.g. here the reference valud is in secs so it would be easy to do the following mistake in two different part of the code

```python
from ._constant import MINUTE
# one part of the code (where delay is expected in secs)
delay = 3 * MINUTES

# other part of the code (where timeout is in uS)
timeout = 3 * MINUTE
```

The latter is wrong. For that reason I would
1. make clear the reference value (e.g. here is trivial `MINUTE = 60  # secs `)
2. keep clear scopes (e.g. in the example above I would duplicate names)